components/nexus-core/src/main/java/org/sonatype/nexus/internal/httpclient/SharedHttpClientConnectionManager.java

```diff
@@ -52,6 +52,8 @@
 
   private final Time connectionPoolIdleTime;
 
+  private final Time connectionPoolEvictingDelayTime;
```

Comment:
being picky: could shorten to `connectionPoolDelayTime` since the field inside the connection pool is `delayMillis` and not `evictingDelayMillis`